v1/nix/pkgs/fetchPipMetadata/src/tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+from os import environ
+import pytest
+import subprocess as sp
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+
+@pytest.fixture()
+def git_repo_path(tmp_path):
+    sp.run(["git", "init"], cwd=tmp_path)
```

Comment:
```suggestion
    subprocess.run(["git", "init"], cwd=tmp_path)
```

Not a fan of import aliasing personally. At least for single usage like here.